packages/kbn-apm-synthtrace/src/scenarios/helpers/distributed_trace_client.ts

```diff
@@ -0,0 +1,189 @@
+/*
```

Comment:
Do you think the helper should live inside `scenarios` folder?

Reply:
Probably not. I'll move it.